### PR TITLE
SIG-Micro proposal

### PR DESCRIPTION
**Pull request by petewarden**

### CODE REVIEW

File: sigs/micro/SIG-request.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## What initial problems will the group tackle?
+
+Prioritize the roadmap for development.
+Divide up high priority work between contributors.
+Share support and help between contributors.
+
+## What modes of communication do you intend to use?
+
+We have a mailing list, will arrange a monthly VC call, and will be asking one of our partners to set up a Slack channel (since the process at Google is involved).
```
Comment (ewilderj) [inline sigs/micro/SIG-request.md]: For the other SIGs, we use Gitter, which has no setup process. e.g. see https://gitter.im/tensorflow/sig-build -- if there's no direct preference to use Slack, we would prefer Gitter. It has no commercial limitations on archiving, etc, and only needs a Github logon, vs creating an admin overhead.

File: sigs/micro/CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# SIG-Micro - TensorFlow Embedded Group
```
Comment (ewilderj) [inline sigs/micro/CHARTER.md]: Preferred "SIG Micro" for consistency with other SIGs.

File: sigs/micro/SIG-request.md (hunk 1 above)
Comment (petewarden) [reply sigs/micro/SIG-request.md]: Thanks, that's helpful, I've set up a chat room at https://gitter.im/tensorflow/sig-micro and updated the proposal.

File: sigs/micro/SIG-request.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Share support and help between contributors.
+
+## What modes of communication do you intend to use?
+
+We have a mailing list, will arrange a monthly VC call, and will be use the Gitter chat channel at https://gitter.im/tensorflow/sig-micro
+
+## Launch plan
+
+
+1.  VC call with initial interested parties to finalize charter and initial group goals.
```
Comment (rajatmonga) [inline sigs/micro/SIG-request.md]: Fix numbering?

Comment (petewarden) [reply sigs/micro/SIG-request.md]: Done, thanks!

**Comment by bhack** [comment]
@ematejska Can you label this with our "RFC Approved"l. It is useful for github stats. 
Thanks

/cc @joanafilipa

**Pull request merged**